docs/spec.openapi.yml

```diff
@@ -0,0 +1,658 @@
+      responses:
+        '200':
+          description: Login succeeded
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResSession'
+        'default':
+          description: An error occurred, see other codes at [Authentication API Spec](https://github.com/hawks-atlanta/authentication-go/blob/main/docs/spec.openapi.yaml)
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResStatus'
+
+                
+  /auth_refresh:
+    post:
+      tags:
+        - Auth
+      security:
+        - bearerAuth: []
+      description: Returns a new token
+      requestBody:
+        content:
+          Object:
+            schema:
+              $ref: '#/components/schemas/Authorization'
+      responses:
+        '200':
+          description: Refresh succeeded
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResSession'
+        'default':
+          description: An error occurred, see other codes at [Authentication API Spec](https://github.com/hawks-atlanta/authentication-go/blob/main/docs/spec.openapi.yaml)
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResStatus'
+
+  # account
+                
+  /account_register:
+    post:
+      tags:
+        - Account
+      description: Creates a new account
+      requestBody:
+        content:
+          Object:
+            schema:
+              $ref: '#/components/schemas/Credentials'
+      responses:
+        '200':
+          description: Register succeeded
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResSession'
+        'default':
+          description: An error occurred, see other codes at [Authentication API Spec](https://github.com/hawks-atlanta/authentication-go/blob/main/docs/spec.openapi.yaml)
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResStatus'
+                
+  /account_password:
+    patch:
+      security:
+        - bearerAuth: []
+      tags:
+        - Account
+      description: Updates the user's password
+      requestBody:
+        content:
+          Object:
+            schema:
+              $ref: '#/components/schemas/ReqAccPassword'
+      responses:
+        '200':
+          description: Password updated successfully
+          content:
+            object:
+              schema:
+                $ref: '#/components/schemas/ResStatus'
+        'default':
+          description: An error occurred, see other codes at [Authentication API Spec](https://github.com/hawks-atlanta/authentication-go/blob/main/docs/spec.openapi.yaml)
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResStatus'
+
+  # file system
```

Comment:
The `/rename` endpoint is missing according to the [Metadata spec](https://github.com/hawks-atlanta/metadata-scala/blob/dev/docs/spec.openapi.yaml)